aws/resource_aws_rds_cluster.go

```diff
@@ -1032,9 +1033,14 @@ func resourceAwsRDSClusterRead(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("error setting vpc_security_group_ids: %s", err)
 	}
 
-	// Fetch and save tags
-	if err := saveTagsRDS(conn, d, aws.StringValue(dbc.DBClusterArn)); err != nil {
-		log.Printf("[WARN] Failed to save tags for RDS Cluster (%s): %s", aws.StringValue(dbc.DBClusterIdentifier), err)
+	tagList, err := conn.ListTagsForResource(&rds.ListTagsForResourceInput{
+		ResourceName: aws.String(aws.StringValue(dbc.DBClusterArn)),
+	})
+	if err != nil {
+		return fmt.Errorf("Failed to get RDS Cluster parameter tags for %s: %s", aws.StringValue(dbc.DBClusterIdentifier), err)
+	}
+	if err := d.Set("tags", keyvaluetags.RdsKeyValueTags(tagList.TagList).IgnoreAws().Map()); err != nil {
+		return fmt.Errorf("[WARN] Failed to save tags for RDS Cluster (%s): %s", aws.StringValue(dbc.DBClusterIdentifier), err)
```

Comment:
Can you please switch this to the `keyvaluetags` package function `RdsListTags`? Thanks!

```go
tags, err := keyvaluetags.RdsListTags(conn, aws.StringValue(dbc.DBClusterArn))

if err != nil {
	return fmt.Errorf("error listing tags for RDS Cluster (%s): %s", aws.StringValue(dbc.DBClusterArn), err)
}

if err := d.Set("tags", tags.IgnoreAws().Map()); err != nil {
	return fmt.Errorf("error setting tags: %s", err)
}
```

Reply:
done